spyderlib/config.py

```diff
@@ -469,11 +470,12 @@ def is_ubuntu():
               # ---- Global ----
               # -- In spyder.py
               '_/close pane': "Shift+Ctrl+F4",
+              '_/lock panes': "Shift+Ctrl+F5",
               '_/preferences': "Ctrl+Alt+Shift+P",
               '_/maximize pane': "Ctrl+Alt+Shift+M",
               '_/fullscreen mode': "F11",
-              '_/toggle next layout': "Shift+Alt+PgDown",
-              '_/toggle previous layout': "Shift+Alt+PgUp",

```

Comment:
I know, but that's exactly the same as removing some options and adding new ones. And the removing part needs a bump in `CONF_VERSION` :-)